### DISCUSSION TITLE
How can I create multiple records at once via JSON POST request?

### DISCUSSION THREAD

**Discussion by stijntersteege** [pocketbase/pocketbase#698] [Q&A] [answered]
Hi there!

I was wondering if it is possible to create multiple records in a collection with one POST request? At the moment it seems I can only POST one JSON item at a time.

Is there a possibility to do so and if not, is that something that might be included in future releases?

Thanks!

**Comment by ganigeorgiev** [comment, marked as answer]
No, currently bulk insertion is not supported.

Transactional bulk create/update/delete will be added in the near future (most likely with #48).

**Reply by stijntersteege** [reply]
Hi Gani,

Thank you for your quick response! Great to hear that this will be possible in the future.
